server/core/src/main/java/org/jolokia/server/core/util/NetworkUtil.java

```diff
@@ -28,7 +30,7 @@
 public final class NetworkUtil {
 
     private static final boolean MAC = System.getProperty("os.name").toLowerCase().contains("mac");
-    private static final List<String> MAC_SKIP_NIFS = List.of("awdl0", "llw0");
+    private static final List<String> MAC_SKIP_NIFS = List.of("awdl", "llw");
```

Comment:
If we go this approach, we can include `utun` here as well. Then a single place where `name.startsWith("utun")` is used should be no longer necessary.

Reply:
What if someone wants to bind Jolokia on `utun`? :)

Reply:
Who wants? :-)